distribution/docker/solr/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM solr:${solr.version}
+
+# Copy solr libs
+COPY maven/jts-core-1.15.0.jar /opt/solr/server/lib/
+
+# Copy solr plugins
+COPY maven/solr-xpath-${project.version}.jar /opt/solr/plugins/
+
+# Copy all solr core configurations to /seed directory
+USER root
+RUN mkdir /seed
+COPY maven/protwords.txt /seed/protwords.txt
+COPY maven/schema.xml /seed/schema.xml
+COPY maven/solrconfig-inmemory.xml /seed/solrconfig-inmemory.xml
+COPY maven/solrconfig.xml /seed/solrconfig.xml
+COPY maven/solr.xml /seed/solr.xml
+COPY maven/stopwords.txt /seed/stopwords.txt
+COPY maven/stopwords_en.txt /seed/stopwords_en.txt
+COPY maven/synonyms.txt /seed/synonyms.txt
+COPY maven/createCore.sh /usr/local/bin/create-ddf-core
+RUN chmod 755 /usr/local/bin/create-ddf-core
+RUN chown -R solr:solr /seed
+USER solr
+
+# Set ENV vars for use by create-ddf-core script
+ENV CONFIG_SRC=/seed
+ENV CORE_DEST=/opt/solr/server/solr/cores
+RUN mkdir -p $CORE_DEST
+RUN /usr/local/bin/create-ddf-core catalog metacard_cache alerts preferences subscriptions
```

Comment:
This is pretty cool.
Downstream projects use additional cores not listed in their tests. Is there a way we can let those projects add more cores without editing any of these files?

Reply:
With this current approach they could package a distro specific docker image by building on top of the one produced here and simply having a `RUN create-ddf-core <cores>` to add the additional cores

I'm already thinking of moving away from that method however and just having an way of creating them dynamically when starting a new solr container. Possibly by passing in a list of core names as an env var. This would the single image more usable by downstreams without having to build a new image on top